iron-meta.html

```diff
@@ -55,12 +55,43 @@
 
   (function() {
 
-    // monostate data
-    var metaDatas = {};
-    var metaArrays = {};
-    var singleton = null;
+    function IronMeta(options) {
+      this.type = (options && options.type) || 'default';
+      this.key = options && options.key;
+    }
```

Comment:
This constructor should probably also handle the `value` property, if provided. `iron-iconset` and `iron-iconset-svg` have lines like this:
```javascript
new Polymer.IronMeta({type: 'iconset', key: this.name, value: this});
```
... just for the sake of putting itself into IronMeta's map. :/ It's easy to work around but I wouldn't be surprised to see it in other places.